command_parse.c

```diff
@@ -1359,81 +1359,58 @@ enum CommandResult parse_subscribe_to(struct Buffer *buf, struct Buffer *s,
 #endif
 
 /**
- * parse_tag_formats - Parse the 'tag-formats' command - Implements Command::parse() - @ingroup command_parse
+ * parse_tag_mapping - Parse a 'tag-formats' or 'tag-transforms' command.
  */
-enum CommandResult parse_tag_formats(struct Buffer *buf, struct Buffer *s,
-                                     intptr_t data, struct Buffer *err)
+static enum CommandResult parse_tag_mapping(struct Buffer *s,
+                                            struct HashTable *ht, const char *what)
```

Comment:
`ht` doesn't describe the hash table's purpose. Could we rename it along the lines of `tag_replacements` to make it obvious?

Similarly `what` could use a better name as it's name is unintuitive without reading its callers. It's datatype could also be constrained. Any objections to use an enum instead?